src/app/cards/components/card-list/card-list.component.ts

```diff
@@ -1,15 +1,17 @@
-import { Component, OnInit } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs';
 
 import { CardsStoreService } from '@/app/cards/services/cards-store.service';
+import { Card } from '@/app/models/card';
 
 @Component({
   selector: 'tmm-card-list',
   templateUrl: './card-list.component.html',
-  styleUrls: ['./card-list.component.scss']
+  styleUrls: ['./card-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
While we're in here, turn on OnPush change detection for all the components since they all rely on observables from the store now.